vanilla-tests/src/test/java/fixtures/url/multi/QueriesTests.java

```diff
@@ -17,17 +18,21 @@ public static void setup() {
 
     @Test
     public void arrayStringMultiNull() {
-        client.getQueries().arrayStringMultiNull(null);
+        try {
+            client.getQueries().arrayStringMultiNull(null);
+            Assert.assertFalse(true); // should not reach here
+        }
+        catch (Exception e) {
+
+        }
```

Comment:
Code seems should not throw exception here.

The definition in swagger is `"x-nullable": true`, so I kind of assume method can take a `null`.
https://github.com/Azure/autorest.testserver/blob/main/swagger/url-multi-collectionFormat.json#L27-L39

Reply:
Good catch @weidongxu-microsoft . The code generated for arrayStringMultiNull() does not support a null parameter. It is trying to stream from a null array and causing an NPE. I will check where the code is generated and modify it to match the Swagger.